packages/terra-form-checkbox/src/Checkbox.jsx

```diff
@@ -83,131 +83,97 @@ const defaultProps = {
   value: undefined,
 };
 
-class Checkbox extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { focus: false };
-    this.onFocus = this.onFocus.bind(this);
-    this.onBlur = this.onBlur.bind(this);
-  }
-
-  onFocus() {
-    this.setState({ focus: true });
-    if (this.props.onFocus) {
-      this.props.onFocus();
-    }
+const Checkbox = ({
+    checked,
+    defaultChecked,
+    inputAttrs,
+    id,
+    isDisabled,
+    isInline,
+    isLabelHidden,
+    labelText,
+    labelTextAttrs,
+    name,
+    onBlur,
+    onChange,
+    onFocus,
+    value,
+    ...customProps
+  }) => {
+  const controlInputAttrs = Object.assign({}, inputAttrs);
+
+  if (checked !== undefined) {
+    controlInputAttrs.checked = checked;
+  } else {
+    controlInputAttrs.defaultChecked = defaultChecked;
   }
 
-  onBlur() {
-    this.setState({ focus: false });
-    if (this.props.onBlur) {
-      this.props.onBlur();
-    }
+  const checkboxClasses = cx([
+    'checkbox',
+    { 'checkbox-inline': isInline },
+    customProps.className,
+  ]);
+
+  const labelClasses = cx([
+    'label',
+    { 'is-disabled': isDisabled },
```

Comment:
I've been out of office, so I apologize for the late post. For html attributes that have a direct correlation to their child (ex: id to id, name to name),  I believe we would just use the attribute name as it is instead of prefixing it with something else. Can we do that here?

Reply:
The prefix is to attach an `is-disabled` css class - instead, are you saying we should change the css class to use a selector: `.native-input:disabled`?

Reply:
Sorry. I meant renaming the isDisabled prop to disabled. I'm fine with the classname and the rest of the implementation.